### DISCUSSION TITLE
Fill Container

### DISCUSSION THREAD

**Discussion by GizmoGoody** [piitaya/lovelace-mushroom#780] [Q&A]
The Fill Container config variable states this - "Fill container or not. Useful when card is in a grid, vertical or horizontal layout."

I'm trying to create a remote control view with mushroom template buttons. Even with Fill Container set to true, the tap action zone seems to only be the center circle around the icon.

Have I missed the point of Fill Container? What is it filling the container with if not the icon and/or action zone?

**Comment by piitaya** [comment]
Yes, I think the card is bigger but not the zone because only the info and the icon is clickable.

**Reply by GizmoGoody** [reply]
@piitaya OK, thanks. What is it filling the container with?
